### PR TITLE
Feature/roc 2476 sending notifications

### PR DESCRIPTION
**Pull request by yarpo**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-2476

### Change description ###
Sending emails to claimant and defendant on rejection or acceptation of offer.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/ClaimStore.java [hunk 1]
```diff
@@ -26,6 +26,10 @@
     @Autowired
     private JsonMapper jsonMapper;
 
+    public Claim getClaim(long claimId) {
```
Comment (rlewan) [inline src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/ClaimStore.java]: Can this method be package scoped?

File: src/test/java/uk/gov/hmcts/cmc/claimstore/events/OfferRespondedCitizenActionsHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    }
+
+    @Test
+    public void shouldSendNotificationsToClaimantWhenOfferAccepted() throws NotificationClientException {
+
+        OfferAcceptedEvent event = new OfferAcceptedEvent(claimWithOffer, MadeBy.CLAIMANT);
+
+        handler.sendNotificationToClaimantOnOfferAcceptedByClaimant(event);
+
+        verify(offerMadeNotificationService).sendNotificationEmail(
```
Comment (geek-soft) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/events/OfferRespondedCitizenActionsHandlerTest.java]: I would consider having that type of test in the claim-store integration tests to make sure notification is triggered on endpoint call.

Comment (yarpo) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/events/OfferRespondedCitizenActionsHandlerTest.java]: I have this test in int tests :)

Comment (yarpo) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/events/OfferRespondedCitizenActionsHandlerTest.java]: Please see `shouldAcceptOfferAndSendNotifications` and `shouldRejectOfferAndSendNotifications`

**Pull request merged**